etc/ReplayOfflineConfiguration.py

```diff
@@ -1021,7 +1053,7 @@
                do_reco=True,
                write_reco=True,
                write_dqm=True,
-               dqm_sequences=["@commonSiStripZeroBias", "@ecal", "@hcal", "@muon", "@jetmet"],
+               dqm_sequences=["@commonSiStripZeroBias", "@ecal", "@hcal", "@muon", "@jetmet", "@L1TMon", "@ctpps"],
```

Comment:
@jfernan2 here the @L1TMon is after @hcal , is that intentional?

Reply:
Uhmm, good catch, it is not intentional.... I guess the replay did not catch it if that dataset was not run.... Thanks!